### ISSUE TITLE
RobotMK => Robotmk

### ISSUE THREAD

**Issue by ypid-geberit** [simonmeggle/robotmk#102]
Sorry that I have to be picky about this but I care about such things. Currently RobotMK and Robotmk is used in writing.

I would propose to define one way to write it and document it. Projects that care about such details often also care about other aspects so even though it is picky, I started to look for such things.

Some good examples:

https://netbox.readthedocs.io/en/stable/development/style-guide/#branding
https://grapheneos.org/

The naming schema of those two projects resonate with me so I would propose RobotMK in writing, otherwise `robotmk` or `RMK` where (Python) naming schema or other conventions take precedence.

PS: I recently gave similar feedback to "CheckMK". The main reason against "Checkmk" or "Robotmk" is that it does not match the way it is pronounced.

**Comment by simonmeggle** [comment]
Hi, 
thanks for the suggestion. I started the project as `RobotMK` as this is the spelling which was the most obvious for me - and which keeps the uppercase "MK" initials. 
I already discussed this with tribe29 who also had a similar decision between check_mk, checkMK, Check_MK etc...  Finally, they came to "Checkmk" and we both agreed that "**Robotmk**" would perfectly fit. 
Long term short: this is WIP :-)
